### PR TITLE
Renaming the tensors and removing tools

### PR DESCRIPTION
**Pull request by tanmayv25**
The changes are being cherry-picked from here: https://github.com/triton-inference-server/tutorials/pull/59

### CODE REVIEW

File: src/model.py [hunk 1]
```diff
@@ -172,17 +174,19 @@ async def generate(self, request):
...
         try:
             request_id = random_uuid()
-            prompt = pb_utils.get_input_tensor_by_name(request, "PROMPT").as_numpy()[0]
+            prompt = pb_utils.get_input_tensor_by_name(
+                request, "text_input"
+            ).as_numpy()[0]
             if isinstance(prompt, bytes):
                 prompt = prompt.decode("utf-8")
-            stream = pb_utils.get_input_tensor_by_name(request, "STREAM").as_numpy()[0]
+            stream = pb_utils.get_input_tensor_by_name(request, "stream").as_numpy()[0]
```
Comment (nnshah1) [inline src/model.py:182]: one other change to be picked up - making stream an optional param:

https://github.com/triton-inference-server/tutorials/pull/58/files

Comment (tanmayv25) [reply src/model.py:182]: Its already included in the changes.

Comment (tanmayv25) [reply src/model.py:182]: https://github.com/triton-inference-server/vllm_backend/blob/40f38ff5c783624bf32027f8d69e77d187e028de/ci/qa_models/vllm_opt/config.pbtxt#L45

**Review by dyastremsky — APPROVED** [review]

**Comment by dyastremsky** [comment]
Great changes. Fantastic work!

File: src/model.py (hunk 1 above)
Comment (nnshah1) [reply src/model.py:182]: @tanmayv25 but don't we need to check here if it is included?

```suggestion
            stream = pb_utils.get_input_tensor_by_name(request, "stream")
            if stream:
               stream = stream.as_numpy()[0]
            else:
               stream = False
```

Comment (tanmayv25) [reply src/model.py:182]: Good catch! I missed it.
Probably one more test case to include in our CI tests.
@pskiran1 ^^^

**Pull request merged**